packages/relay/src/services/PostService.ts

```diff
@@ -183,11 +150,11 @@ export class PostService {
 
         return await Promise.all(
             posts.map(async (post) => {
+                // TODO should remove loop sql query and replace it as join
```

Comment:
Note: Issue to be opened.

Reply:
@Lisooo790926 Can you create a new issue for this, rebase the PR and feel free to merge it.